index.html

```diff
@@ -49,5 +53,6 @@
     </ul>
   </nav>
 </div>
+<script src="script.js"></script>
```

Comment:
If you're going to use jQuery, you have to include it in your HTML. You can use a CDN or load it yourself, but it has to be there.